src/plugins/mer/merbuildconfigurationaspect.cpp

```diff
@@ -120,6 +134,84 @@ class MerBuildConfigurationWidget : public ProjectExplorer::NamedWidget
         vbox->addWidget(m_detailsContainer);
 
         updateSummary();
+
+        auto setSignElementsEnabled = [](Ui::MerBuildConfigurationWidget *ui, bool enabled) {
+            ui->signingUserChangeButton->setEnabled(enabled);
+            ui->signingUserLineEdit->setEnabled(enabled);
+            ui->signingPassphraseFileChooser->setEnabled(enabled);
+            ui->signingUserClearButton->setEnabled(enabled);
+        };
+
+        m_ui->signPackagesCheckBox->setText(tr(Constants::MER_SIGN_PACKAGES_OPTION_NAME));
+        m_ui->signingUserClearButton->setIcon(Utils::Icons::EDIT_CLEAR.icon());
+
+        m_ui->signPackagesCheckBox->setChecked(m_aspect->signPackages());
+        QString gpgErrorString;
+        const bool isGpgAvailable = Sfdk::isGpgAvailable(&gpgErrorString);
+        if (!isGpgAvailable) {
+            m_ui->signErrorLabel->setText(gpgErrorString);
+            m_ui->signErrorIconLabel->setPixmap(Icons::CRITICAL.pixmap());
```

Comment:
Wasn't the intention to let it look less critical? :) Maybe Icons::WARNING would be enough?